app/views/dashboard/index.html.erb

```diff
@@ -15,7 +15,7 @@
             <div class='widget_control'>
               <a class='remove' data-url='<%= widget_path(widget) %>'>&times;</a>
             </div>
-            <div class="widget <%= widget.name.parameterize %>">
+            <div class="widget <%= (widget.data[:settings][:class_name] || widget.name).parameterize %>">
```

Comment:
this will not work and fail if data does not have :settings key (first thing that I got 500 for when I checkout this PR and enter dashboard)